src/Geta.404Handler/Views/NotFoundRedirect/Redirects.ascx

```diff
@@ -66,6 +71,12 @@
                 <td class="shorter delete">
                     <input name="skipWildCardAppend" type="checkbox" />
                 </td>
+                <td class="shorter">
+                    <select name="redirectType" id="redirectType">
+                        <option value="301">301</option>
```

Comment:
In the user interface it would be better to use textual values - Permanent and Temporal. Then should add appropriate translations too.

Reply:
Textual translation added